code/game/objects/items/weapons/cigs.dm

```diff
@@ -95,6 +95,17 @@ LIGHTERS ARE IN LIGHTERS.DM
 	else if(istype(W, /obj/item/device/assembly/igniter))
 		light("<span class='notice'>[user] fiddles with [W], and manages to light their [name].</span>")
 
+	else if(istype(W, /obj/item/weapon/gun/magic/wand/fireball))
+		var/obj/item/weapon/gun/magic/wand/fireball/F = W
+		if(F.charges)
+			if(prob(50))
+				light("<span class='notice'>Trying painfully hard to be cool, [user] manages to light their [name] with [W], singing their eyebrows.</span>")
+			else
```

Comment:
This should never happen if you're a wizard.

They're too....magically-ish for this to happen.